### DISCUSSION TITLE
Couldn't understand how to use the plug in

### DISCUSSION THREAD

**Discussion by AviramZivGTH** [ryanluker/vscode-coverage-gutters#326] [Q&A]
I've installed the extension to my vs-code.
But when I'm running my tests nothing happens. (I mean I don't see any inline coverage at all)
I understand that I should have an lcov style coverage file on my project, but I don't have it by default and 
I didn't understand how to create\generate my own one. I believe the plugin somehow relay on this file.
My Projects are JAVA written in JAVA with TestNG (and not JUnit) so if there a specific way I should use I would love to get an instructions about it.
I used to work with eclipse and the plugin coverage there was "plug and play"....

Thanks!

**Comment by ryanluker** [comment]
@AviramZivGTH Thanks for submitting a discussion item!
Hmm sounds like you need a way to generate a coverage file for your java project. I assume you have looked at the example java project?
https://github.com/ryanluker/vscode-coverage-gutters/tree/master/example/java

In terms of what coverage file to generate, the system can consume a variety of types and file options (lcov, xml, etc).
https://stackoverflow.com/a/41664305

I believe jacoco is the most popular amongst java devs? See the link above with some tactics for generating this file, let me know if this gets you any closer!

**Comment by ssprojecttest** [comment]
I am having similar issues but I am using dotnet project. Also,  not sure how to create/generate lcov file? could someone please help me getting these information.

**Reply by ryanluker** [reply]
@ssprojecttest I answered in a different discussion item but here is the example project for dotnet https://github.com/ryanluker/vscode-coverage-gutters/tree/master/example/dotnet put together by @flatrick-priv
